### DISCUSSION TITLE
Audit Log

### DISCUSSION THREAD

**Discussion by M1scer** [firefly-iii/firefly-iii#7963] [Firefly III]
Hi,

i am using tags to organize my transactions. The Budget and/or category are assigned according to the tag.
To assign the budget and category i have to run a cronjob executing the rules. This works fine.

Now i regocnized that it will always create a new audit log entry, each time the cronjob is executed.
That means even if the category or budget is already assigned and the rule will assign the same budget or category it will create a new entry.
This leads to very long audit logs.

I think it would be good to have a audit log entry only if there is a real change.

A workaround for me would be to disable the audit log, is there a option to do this?

![grafik](https://github.com/firefly-iii/firefly-iii/assets/93549967/4fb8ec33-8bb8-486c-9540-494420bc547e)

**Comment by JC5** [comment]
I'll check it out 👍
